.readthedocs.yml

```diff
@@ -2,5 +2,8 @@ requirements_file: dev_tools/conf/pip-list-dev-tools.txt
 build:
```

Comment:
`requirements_file: dev_tools/conf/pip-list-dev-tools.txt` shouldn't be needed with the addition of `dev_env` to `extra_requirements`